.gitignore

```diff
@@ -3,6 +3,7 @@ build.gradle
 
 # IntelliJ
 *.iml
+.idea/
```

Comment:
just occurred to me why you have .idea in this folder. In the usual setup IntelliJ is setup with Terasology so the .idea folder is in the root of the Terasology project not the indvidual module.